common/redirects/ImplementationValidator.html

```diff
@@ -0,0 +1,11 @@
+---
+title:
+ ImplementationValidator
+permalink:
+ ig/validator.pack
```

Comment:
Based on our past experience, this one will need an Akamai change to support the disconnect between `.html` and `.pack`.  Are you coordinating those as part of this ticket?